src/app/api-explorer-jsviewer.ts

```diff
@@ -36,13 +37,13 @@ export function initializeJsonViewer() {
 }
 
 export function initializeResponseHeadersViewer() {
-    let jsonViewer = getAceEditorFromElId("response-header-viewer");
-    let ariaLabel = getTextAreaAriaLabel("Response headers viewer");
+    const jsonViewer = getAceEditorFromElId('response-header-viewer');
```

Comment:
Why did you decide to change all [let]s to [const]s?

Reply:
Not all  variables declared with let were changed to const only those that are never reassigned.

This helps in reducing cognitive overload when reading through the code. How? When you see a variable declared with const you know that it won't be reassigned down the line.